docs/HISTORY.txt

```diff
@@ -9,7 +10,8 @@ Breaking changes:
 
 New features:
 
-- *add item here*
+- More Python 2 / 3 compatibility adjustments
```

Comment:
Personally, I am filing these under the `Bug fixes` heading instead of `New features`. It's a bit of both. On one hand the package does nothing new, so it is no new feature. On the other hand the package gains Python 3 compatibility, which you can call a feature. Oh well.

Reply:
Yeah, @jensens used almost the same words as your just a few days ago in another PR.

I do empathise more though with the idea that compatibility with a newer version of Python is indeed a new feature, much more than a bug fix. I mean, it's not a bug to get old. Oh well.

Reply:
Oh well...... ;)